src/Database/Bloodhound/Client.hs

```diff
@@ -495,9 +497,9 @@ getNodesStats sel = parseEsResponse =<< get =<< url
   where
     url = joinPath ["_nodes", selectionSeg, "stats"]
     selectionSeg = case sel of
-      LocalNode -> "_local"
+      LocalNode          -> "_local"
```

Comment:
In future please leave whitespace/formatting changes out of a PR that makes semantic changes to the library.

Reply:
Not necessarily opposed to such changes, but they need to be proposed and PR'd separately.